src/app/about/page.client.tsx

```diff
@@ -0,0 +1,151 @@
+'use client';
+import TeamBioCard from '@/components/team-bio-card/team-bio-card';
+import {
+  Box,
+  Button,
+  Pagination,
+  PaginationItem,
+  Typography,
+} from '@mui/material';
+import Container from '@mui/material/Container';
+import Script from 'next/script';
+import React, { useState } from 'react';
+import './about.scss';
+
+interface TeamMember {
+  name: string;
+  job_title: string;
+  bio?: string;
+  linkedin_link: string;
+  github_link?: string;
+  role: string[];
+  profile_image_link: string;
+  brief_descriptions?: string;
+}
+
+interface AboutPageProps {
+  teamMemberData: TeamMember[];
+}
+
+export default function About({ teamMemberData }: AboutPageProps) {
+  const [filter, setFilter] = useState<string>('All Members');
```

Comment:
To make this code more maintainable, I think we should do:

```suggestion
  const [filter, setFilter] = useState<string>(roles[0]);
```

This ensures that there are fewer places where the string literal needs to be changed, should we ever change what `All Members` might do